client/knowledgebase/components/CategoryDetail.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+import Ionicons from 'react-ionicons';
+import { Articles } from '../components';
+import { ItemMeta } from '../components';
+
+export default class CategoryDetail extends React.Component {
```

Comment:
If current component is stateless it would be better to write function component.

Reply:
done